### PR TITLE
Add nmea_serial_driver launch file

### PR DESCRIPTION
**Pull request by diasdm**
Regarding issue #49

### CODE REVIEW

File: launch/nmea_serial_driver.launch [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  <!-- A simple launch file for the nmea_serial_driver node. -->
+
+  <arg name="port" default="/dev/ttyUSB0" />
+  <arg name="baud" default="4800" />
+  <arg name="frame_id" default="gps" />
+  <arg name="time_ref_source" default="gps" />
+  <arg name="useRMC" default="False" />
+
+  <node name="garmin35lp_nmea_node" pkg="nmea_navsat_driver" type="nmea_serial_driver" output="screen">
```
Comment (evenator) [inline launch/nmea_serial_driver.launch]: Would you mind changing the node name to something more generic?

Comment (diasdm) [reply launch/nmea_serial_driver.launch]: My bad, done

**Comment by diasdm** [comment]
Done. No problem!

**Comment by evenator** [comment]
Thanks for your contribution!

**Pull request merged**